rke/templates/templates.go

```diff
@@ -157,7 +158,8 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 		},
 		kdm.MetricsServer: {
 			">=1.23.3-rancher1-1":                     metricsServerv061,
-			">=1.20.14-rancher2-1 <1.23.3-rancher1-1": metricsServerv050,
+			">=1.22.14-rancher1-1 <1.23.3-rancher1-1": metricsServerv051,
+			">=1.20.14-rancher2-1 <1.22.13-rancher1-1": metricsServerv050,
```

Comment:
These ranges exclude `1.22.13-rancher1-1` altogether meaning that there is no template for `1.22.13-rancher1-1` and it would get broken.  Also, we're no longer introducing `1.22.14-rancher1-1` so I recommend the following instead:
```suggestion
			">=1.22.15-rancher1-1 <1.23.3-rancher1-1": metricsServerv051,
			">=1.20.14-rancher2-1 <=1.22.13-rancher1-1": metricsServerv050,
```

Reply:
Ah! right. Thanks for the catch.